### ISSUE TITLE
[Feature Request] Ensure that every track is a clean rec + a long loop

### ISSUE THREAD

**Issue by spudpiggy** [corbindavenport/wii-shop-extension#146]
TL;DR Please fix the 3DS eShop and DSi Shop tracks, as I find these recordings hard to use

I find myself only using the Wii Shop option because the other options are either short loops or bad recordings.
For example, the 3DS eShop track (/the June 2011 track, as the eShop used to cycle its music) has menu noises still present in the recording.
This is distracting enough that I straight up cannot use it.

The DSi Shop track is a better capture, but it's a short loop, only a couple minutes or so.
Tracks restarting during a long browsing session is kinda annoying, enough so to be a dealbreaker to me.
I recommend you use this [30 minute loop recorded by @ToruTheRedFox](https://www.youtube.com/watch?v=lttQ47yZ-5A) instead, once he gives permission ofc.
I don't know where to find a longer loop of the (June 2011) eShop track though, although it could be ripped/captured again if needed. Anything above 15 minutes would be okay, but 30 is probably better.

**Comment by spudpiggy** [comment]
READ EDIT 2, I REALISED OGGS CAN LOOP SO YOU'D ONLY HAVE TO MAKE THE LOOPING FILES!
much less effort than i figured itd be lol
